Packs/JSONSampler/Integrations/JSONSampler/JSONSampler.yml

```diff
@@ -0,0 +1,51 @@
+category: Utilities
+commonfields:
+  id: JSONSampler
+  version: -1
+configuration:
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- defaultvalue: "1"
+  display: Incidents Fetch Interval
+  name: incidentFetchInterval
+  required: false
+  type: 19
+- display: The raw JSON string to use as the sample data
+  name: JSON
+  required: true
+  type: 12
+- display: The incident name to give to the created incident
+  name: name
+  required: false
+  type: 0
+description: |-
+  A utility for testing incident fetching with mock JSON data.
+display: JSONSampler
+name: JSONSampler
+script:
+  commands:
+  - arguments:
+    - description: The key to change.  Must also set value arguement.  Can be comma
+        separated to change multiple values.
+      name: key
+    - description: The new key value. Must also set key argument. Can be comma separated
+        to support changing multiple values.
+      name: value
+    description: Sample the provided JSON and return the results to the Context and
+      Warroom.  Can use key and value arg to change a JSON values if desired.
+    name: sampler
```

Comment:
in order to not be too generic, lets modify the command name to be more specific with the integration name as prefix and `-` as separator

Reply:
Updated this to be more in line with other integrations as suggested.